cmake/CatkinBuild.cmake

```diff
@@ -0,0 +1,72 @@
+# Set compiler flags
+set(CMAKE_CXX_STANDARD 14)
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+if (NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
+  add_definitions(-O3)
+endif(NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
+
+# Find catkin macros and libraries
+find_package(catkin REQUIRED)
+find_package(Eigen3 REQUIRED)
+find_package(Boost REQUIRED COMPONENTS python)
```

Comment:
Only when Python bindings are built?

Reply:
Hmm, but isn't this find_package anyways done inside python/CMakeLists.txt? It might be superfluous here...